plugins/outputs/cloudrun/cloudrun.go

```diff
@@ -0,0 +1,197 @@
+package cloudrun
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+	"github.com/influxdata/telegraf/config"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/common/gcp"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
+const (
+	defaultURL = "http://127.0.0.1:8080/telegraf"
+)
```

Comment:
Maybe move this to the other `default` constants below?
```suggestion
const defaultURL = "http://127.0.0.1:8080/telegraf"
```